requirements.txt

```
@@ -1,2 +1,3 @@
 clickhouse-driver~=0.2.0
 apache-airflow>=2.0.0,<2.5.0
+apache-airflow-providers-common-sql
```

Looks like the package is installed every time: https://github.com/bryzgaloff/airflow-clickhouse-plugin/actions/runs/3567733669/jobs/6003994155#step:5:346

So lets' pin its version to the latest one:

```suggestion
apache-airflow-providers-common-sql==1.3.0
```

---

Not sure if it's a good decision as official docker images apache/airflow:2.4.x have v1.2.0